frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -70,33 +68,32 @@ export const detectRGW: FeatureDetector = async (dispatch) => {
 
 export const detectOCS: FeatureDetector = async (dispatch) => {
   try {
-    const storageCluster = await fetchK8s(
-      OCSServiceModel,
-      OCS_INTERNAL_CR_NAME,
-      CEPH_STORAGE_NAMESPACE,
-    );
+    const storageClusters = await k8sList(OCSServiceModel, { ns: CEPH_STORAGE_NAMESPACE });
+    const storageCluster = storageClusters?.[0];
```

Comment:
The one with the not Ignored needs to be checked. Because there will be only one in that case.

I have seen other dashboards getting enabled when 0th index is picked.